src/makie.jl

```diff
@@ -587,6 +587,9 @@ end
 
 function mixing_surface_contour3d_contour_and_contourf()
     GLMakie.activate!() # hide
+    url = "https://raw.githubusercontent.com/JuliaImages/TestImages.jl/images/images/coffee.png"
```

Comment:
Is this link going to be static? Isn't it better for us to use a image in our repo? We could add the image if the license permits to our repo.

Reply:
is static, as far as I know. But, yes, I think we can have it in our site as well... I could do it in the next iteration.

Reply:
Why not
```
using TestImages
img = testimage("coffee.png")
```?

Reply:
I was under the impression that Images, TestImages is a huge dependency.... not sure if that true nowadays.

Reply:
But, if you guys are ok with that, I would prefer it this way 😄 , using TestImages is definitely better.

Reply:
@rikhuijzer is the criteria for new dependencies/files. If he is ok with it, them I'm also ok.

Reply:
Yeah sounds good. Adding images will increase repository size (almost) irreversibly, but adding a package can easily be reversed.